csr/csr.go

```diff
@@ -95,6 +97,25 @@ func VerifyCSR(csr *x509.CertificateRequest, maxNames int, keyPolicy *goodkey.Ke
 	if len(badNames) > 0 {
 		return fmt.Errorf("policy forbids issuing for: %s", strings.Join(badNames, ", "))
 	}
+	if err := wildcardOverlap(names); err != nil {
+		return err
+	}
+	return nil
+}
+
+// matchesWildcard takes a map of domain names and returns an error if any of
```

Comment:
This comment refers to `matchesWildcard` but it looks like you renamed the function to `wildcardOverlap`